main.py

```diff
@@ -1,11 +1,30 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
-
-
-if __name__ == "__main__":
-    main()
+    with open("players.json") as players_data:
+        players_info = json.load(players_data)
+    for player_name, player_data in players_info.items():
+        player = Player(
+            nickname=player_name,
+            email=player_data["email"],
+            bio=player_data["bio"],
+            race=Race.objects.get_or_create(
+                name=player_data["race"]["name"],
+                defaults={"description": player_data["race"]["description"]}
+            )[0],
+            guild=Guild.objects.get_or_create(
+                name=player_data["guild"]["name"],
+                defaults={"description": player_data["guild"]["description"]}
+            )[0] if player_data.get("guild") else None,
+        )
+        for skill_data in player_data["race"]["skills"]:
+            Skill.objects.get_or_create(
+                name=skill_data["name"],
+                defaults={"bonus": skill_data["bonus"], "race": player.race}
+            )
+        player.save()
```

Comment:
The `created_at` field in the `Player` model will be updated every time the player is saved due to `auto_now=True`. If you want it to be set only once at the time of creation, consider using `auto_now_add=True`.